packages/lesswrong/lib/routes.js

```diff
@@ -316,6 +316,16 @@ addRoute([
   },
 ]);
 
+addRoute([
+  // GreaterWrong comment (mostly for use with hover previews)
+  {
+    path:'/posts/:_id/:slug/comment/:comment_id?',
+    name: 'comment.greaterwrong',
+    componentName: "PostsSingle",
+    previewComponentName: "PostCommentLinkPreviewGreaterWrong",
```

Comment:
What happens if you go to this route directly? If I'm reading correctly, it'll give you the post, but without the comment-permalink part, and without the page title/subtitle being set correctly. Ie, this should probably have
```
    titleComponentName: 'PostsPageHeaderTitle',
    subtitleComponentName: 'PostsPageHeaderTitle',
```